tumblr-photo-video-ripper.py

```diff
@@ -187,16 +185,17 @@ def _download_media(self, site, medium_type, start):
             os.mkdir(target_folder)
 
         base_url = "https://{0}.tumblr.com/api/read?type={1}&num={2}&start={3}"
+        retry_times = 0
```

Comment:
I think I've added retrying somewhere.

Reply:
This is retry logic for each tumblr blog. This addresses the infinite loop issue where tumblr gives back an unexpected response so we can't parse it correctly. If that happens RETRIES number of times in a row then we stop trying to access that blog and move on to the next one. The infinite loop issue that you had before was the result of continuing the loop without advancing in the event of an exception.

Here we apply the same retry logic that we do for individual posts, ie. if we fail to parse the data returned by tumblr RETRIES times then we move on to the next url for that blog, if that again fails RETRIES times then we move on to the next site.